.gitignore

```diff
@@ -130,3 +130,4 @@ dmypy.json
 .idea
 CarShowRoom/.idea
 env/db.env
+CarShowRoom/setup.cfg
```

Comment:
Because there are two similar files, i used this one to copy it into docker container so i could check code style inside of container(using its terminal). File setup.cfg is also located in outer directory, that is not ignored by git